2015-03-13 09:48:29 LOG_ERROR: 30
2015-03-13 09:48:29 LOG_ERROR: 35
2015-06-17 17:01:49 LOG_ERROR: 
FusionConvConverterCloudooo::_xmlRpc2Convert() - APP/Plugin/FusionConv/Utility/Converter/FusionConvConverterCloudooo.php, line 159
FusionConvConverterCloudooo::convert() - APP/Plugin/FusionConv/Utility/Converter/FusionConvConverterCloudooo.php, line 182
FichesController::genereFusion() - APP/Controller/FichesController.php, line 336
ReflectionMethod::invokeArgs() - [internal], line ??
Controller::invokeAction() - CORE/Cake/Controller/Controller.php, line 490
Dispatcher::_invoke() - CORE/Cake/Routing/Dispatcher.php, line 185
Dispatcher::dispatch() - CORE/Cake/Routing/Dispatcher.php, line 160
[main] - APP/webroot/index.php, line 108
'Erreur du serveur Cloudooo "http://127.0.01:8011/" (code 0): HTTP_Request2_ConnectionException: Unable to connect to tcp://127.0.01:8011. Error: Connection refused in /usr/share/php/HTTP/Request2/Adapter/Socket.php on line 325'
